modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/moth.dm

```diff
@@ -1,25 +1,13 @@
 /datum/species/moth
-	mutant_bodyparts = list()
 	default_mutant_bodyparts = list(
 		"fluff" = "None",
 		"wings" = ACC_RANDOM,
 		"moth_antennae" = ACC_RANDOM
 	)
-	species_traits = list(
-		LIPS,
-		NOEYESPRITES,
-		HAS_FLESH,
-		HAS_BONE,
-		HAS_MARKINGS,
-		MUTCOLORS
-	)
-	inherent_traits = list(
-		TRAIT_ADVANCEDTOOLUSER,
-		TRAIT_CAN_STRIP,
-		TRAIT_CAN_USE_FLIGHT_POTION,
-	)
-	learnable_languages = list(/datum/language/common, /datum/language/moffic)
-	payday_modifier = 0.75
+
+/datum/species/moth/New()
+	. = ..()
+	species_traits |= MUTCOLORS
```

Comment:
Will this change fix all of the stuff for the moths? Just making sure we don't lose anything here, like for instance the languages they can learn (which I guess are stock-tg ones, but still)

Reply:
At the moment, any species can learn any language, the removal was part me feeling it was obsolete, a relic from the old prefs? If I am wrong, I can return those.

Reply:
It's more so they start with the language without having to select it themselves, IIRC.